platform/ios/PolyPodApp/PodApi/PostOffice.swift

```diff
@@ -434,21 +434,33 @@ extension PostOffice {
 }
 
 extension PostOffice {
-    private func handleNetwork(method: String, args: [Any], completionHandler: @escaping (MessagePackValue?, MessagePackValue?) -> Void) {
+    private func handleEndpoint(method: String, args: [Any], completionHandler: @escaping (MessagePackValue?, MessagePackValue?) -> Void){
         switch method {
-        case "httpPost":
-            handleNetworkHttpPost(args: args, completionHandler: completionHandler)
-        default:
-            Log.error("PolyNav method unknown: \(method)")
+        case "send":
+            handleEndpointSend(args: args, completionHandler: completionHandler)
+        case "get":
+            handleEndpointGet(args: args, completionHandler: completionHandler)
+        default: Log.error("PolyEndpoint method unknown: \(method)")
         }
     }
     
-    private func handleNetworkHttpPost(args: [Any], completionHandler: @escaping (MessagePackValue?, MessagePackValue?) -> Void) {
-        let url = args[0] as! String
-        let body = args[1] as! String
+    private func handleEndpointSend(args: [Any], completionHandler: @escaping (MessagePackValue?, MessagePackValue?) -> Void) {
```

Comment:
Every component of `args` is a `String`. Can you maybe define it here as `[String]`?